Lib/urllib/parse.py

```diff
@@ -38,29 +39,36 @@
            "DefragResult", "ParseResult", "SplitResult",
            "DefragResultBytes", "ParseResultBytes", "SplitResultBytes"]
 
-# A classification of schemes ('' means apply by default)
-uses_relative = ['ftp', 'http', 'gopher', 'nntp', 'imap',
+# A classification of schemes.
+# We use '' as the scheme value for default scheme in urlsplit and urlparse.
+
+uses_relative = ['', 'ftp', 'http', 'gopher', 'nntp', 'imap',
```

Comment:
Uses_relative only seems to affect “urljoin”, not “urlsplit” nor “urlparse”. The empty string means that you can join one relative URL onto another one.